index.js

```diff
@@ -345,6 +345,13 @@
     function(typeRep) { return []; }
   );
 
+  //  Ordering :: Type
+  var Ordering = $.EnumType(
+    'sanctuary/Ordering',
+    readmeUrl('Ordering'),
+    ['LT', 'EQ', 'GT']
+  );
```

Comment:
I'm certainly in favour of documenting `Ordering`.

If we're to export `LT`, `EQ`, and `GT` it's probably a good idea to make their values `-1`, `0`, and `1` respectively for more convenient interoperability with `Array#sort`.